README.md

```diff
@@ -22,7 +22,7 @@ Vitess Operator Version | Recommended Vitess Versions | Recommended Kubernetes V
 `v2.3.*` | `v9.0.*`                    | `v1.15.*`, `v1.16.*`, or `v1.17.*`
 `v2.4.*` | `v10.0.*`                   | `v1.15.*`, `v1.16.*`, or `v1.17.*`
 `v2.5.*` | `v12.0.*`                   | `v1.17.*`, `v1.18.*`, or `v1.19.*`
-`v2.6.*` | `v13.0.*`                   | `v1.20.*`, `v1.21.*`, or `v1.22.*`
+`v2.6.*` | `v12.0.*`& `v13.0.*`        | `v1.20.*`, `v1.21.*`, or `v1.22.*`
```

Comment:
Sorry to be nitpicky, but should we do `or` to be consistent with how we are listing k8s versions?

Reply:
no problem, done